tests/Models/TestSoftDeletableModel.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Tests\Models;
+
+use Closure;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Spatie\ModelCleanup\CleanupConfig;
+use Spatie\ModelCleanup\GetsCleanedUp;
+
+class TestSoftDeletableModel extends Model implements GetsCleanedUp
```

Comment:
I'm thinking we could extend the other test model. This way we only have to add `SoftDeletes` and set the table name.